### PR TITLE
Add training and guides to SCO skeleton page

### PR DESCRIPTION
**Pull request by hughes-dustin**
## Page to edit
url: https://staging.va.gov/education/school-resources/

## Description of what's needed (edits/link changes/additions)
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19889

Adding in 'Training and guides' content.

### CODE REVIEW

**Comment by zurbergram** [comment]
Look at https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/components/merger-social.html for accordion classes to use

File: pages/education/school-resources.md [hunk 1]
```diff
@@ -138,23 +138,118 @@ Resources for schools is a one-stop shop for School Certifying Officials (SCOs)
...
+                    <b>[School Official’s Handbook >](https://www.benefits.va.gov/GIBILL/docs/job_aids/SCO_Handbook.pdf)</b>     
+                    <br><b>[VA-ONCE quick reference guide >](https://www.benefits.va.gov/GIBILL/docs/vaonce/VAONCEguide.pdf)</b>  
+                    <br><b>[Find your Education Liaison Representative (ELR) >](https://www.benefits.va.gov/gibill/resources/education_resources/school_certifying_officials/elr.asp)</b>   
+                    <br><b>[Training webinars >](https://www.benefits.va.gov/gibill/resources/education_resources/school_certifying_officials/presentations.asp)</b>     
+                    <br><b>[WEAMS Public >](https://inquiry.vba.va.gov/weamspub/buildSearchInstitutionCriteria.do)</b>     
+                    <br><b>[Benefits A to Z >](https://benefits.va.gov/atoz/)</b>   
+                    <br><b>[Check your status as a Covered Institution >](https://www.benefits.va.gov/GIBILL/resources/education_resources/school_certifying_officials/Covered_Institutions.asp)</b>
+                    <br><br>
+                    <b>Reporting:</b>
+                    <ul>
```
Comment (shawkey-daniel) [inline pages/education/school-resources.md]: Can you just use a nested list here? Also, as Kenny pointed out, the normal markdown link here is underlined, but the prototype has non-underlined links, so may need to use html with css.

File: pages/education/school-resources.md [hunk 2]
```diff
@@ -138,23 +138,118 @@ Resources for schools is a one-stop shop for School Certifying Officials (SCOs)
...
+                    <br><b>[VA-ONCE P042 - Training on enhancements in newest version >](https://www.benefits.va.gov/gibill/docs/vaonce/VA-ONCE_P042_Training.ppt)</b>     
+                    <br><b>[User tips for VA-ONCE >](https://www.benefits.va.gov/gibill/docs/vaonce/va-once_tips.pdf)</b>   
+                    <br><b>[Automation of Post-9/11 GI Bill benefits and VA-ONCE remarks >](https://www.benefits.va.gov/GIBILL/docs/letters/Automation_of_Post911_GI_Bill_and_VAONCE_Remarks.pdf)</b>     
+                    <br><b>[Top ten list of what not to report in VA-ONCE >](https://www.benefits.va.gov/GIBILL/docs/vaonce/Tips_for_VA-ONCE_Remarks.pdf)</b>  
+                    <br><b>[Colmery extension campus update webinar (July 2019) >](https://www.benefits.va.gov/GIBILL/docs/SCO/Ext_Campus_SCO_Virtual_Webinar.pdf)</b>   
+                    <br>
+                    <b>Extension campus certification scenarios:</b>
```
Comment (shawkey-daniel) [inline pages/education/school-resources.md]: Same as comment above.

File: pages/education/school-resources.md [hunk 3]
```diff
@@ -138,23 +138,118 @@ Resources for schools is a one-stop shop for School Certifying Officials (SCOs)
...
+            <li>
+                <button class="usa-accordion-button usa-button-unstyled" aria-controls="t_3">
+                    Payment and debt              
+                </button>     
+                <div id="t_3" class="usa-accordion-content">
+                <b>[Payment and debt information resources >](https://www.benefits.va.gov/gibill/resources/education_resources/school_certifying_officials/payment_info.asp)       
+                <br>[Direct deposit information for schools >](https://gibill.custhelp.va.gov/app/answers/detail/a_id/1428/~/how-can-my-school-enroll-in-ach-direct-deposit%2C-or-change-its-ach-direct)      
+                <br>[School overpayments and using Dual Certification Process to avoid them >](https://www.benefits.va.gov/GIBILL/docs/DualCertificationTips.pdf)      
+                <br>[The Treasury Offset Program: How to receive your state and federal funding >](https://www.benefits.va.gov/gibill/docs/job_aids/Treasury_TOP_Q_A.pdf)  
+                <br>[Payment fees updated with correct tuition and fees caps (9/17/2018) >](https://www.benefits.va.gov/GIBILL/docs/fgib/107_TandF_SCO_letter.pdf)</b>
```
Comment (shawkey-daniel) [inline pages/education/school-resources.md]: Same as comment above.

File: pages/education/school-resources.md (hunk 1 above)
Comment (hughes-dustin) [reply pages/education/school-resources.md]: change has been pushed.

File: pages/education/school-resources.md (hunk 2 above)
Comment (hughes-dustin) [reply pages/education/school-resources.md]: change has been pushed.

File: pages/education/school-resources.md (hunk 3 above)
Comment (hughes-dustin) [reply pages/education/school-resources.md]: change has been pushed.

**Comment by hughes-dustin** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

**Pull request merged**